data.json

```diff
@@ -641,11 +650,11 @@
   {
     "name": "Uno",
     "difficulty": "easy",
-    "description": "Register for the Hacktoberfest event. Share your contact details. Make a pull-request to improve the projects code or documentation.",
-    "reference": "https://platform.uno/uno-is-joining-hacktoberfest-2019/",
+    "description": "Register for the Hacktoberfest event. Make a valid PR and fill up a <a href='https://www.surveymonkey.com/r/FNDG269'>form</a>",
```

Comment:
As stated in #754, we won't list this opportunity this year. Please reset you're changes

Reply:
Ok, removing Uno & Correcting date format